src/formatters/networkFormatter.ts

```diff
@@ -37,3 +41,61 @@ export function getFormattedHeaderValue(
   }
   return response;
 }
+
+export async function getFormattedResponseBody(
+  httpResponse: HTTPResponse,
+  sizeLimit: number = BODY_CONTEXT_SIZE_LIMIT,
+): Promise<string> {
+  try {
+    const responseBuffer = await httpResponse.buffer();
+
+    if (isUtf8(responseBuffer)) {
+      const responseAsTest = responseBuffer.toString('utf-8');
+
+      if (responseAsTest.length === 0) {
+        return `<empty response>`;
+      }
+
+      return `${getSizeLimitedString(responseAsTest, sizeLimit)}`;
+    }
+
+    return `<binary data>`;
+  } catch {
+    // buffer() call might fail with CDP exception, in this case we don't print anything in the context
+    return '';
```

Comment:
Let's just make this return type `string|undefined`  and we can remove the wrappers in the MCPResponse.
I think in the case the Body is empty we still want to show that it was empty as long as the above succeeded.